driver/examples/configs/baroclinic_c12_orch_cpu.yaml

```diff
@@ -0,0 +1,89 @@
+stencil_config:
+  compilation_config:
+    backend: dace:cpu
+    rebuild: false
+    validate_args: true
+    format_source: false
+    device_sync: false
+initialization:
+  type: baroclinic
+performance_config:
+  performance_mode: true
+  experiment_name: c12_baroclinic
+nx_tile: 12
+nz: 79
+dt_atmos: 225
+minutes: 60
```

Comment:
[optional] run it for less than time since the purpose is to make sure it builds and run